config/ftbquests/quests/chapters/tools.snbt

```diff
@@ -3018,55 +3019,76 @@
 			x: -4.5d
 			y: -15.5d
 			subtitle: "Close enough"
-			description: ["First tier of jetpacks are made from wood, stone and copper. Wood is the basic one, needed for anything better, second is stone and the best is copper!"]
+			description: [
+				"First tier of jetpacks are made from very common materials, what makes them great for early game flying!"
+				""
+				"Wooden Jetpack (Tier 0) is the starting point of your jetpack upgrade path. There is 5 tiers of jetpacks in total, and each tier requires a jetpack from the lower tier for crafting."

```

Comment:
`Wooden Jetpack (Tier 0) is the starting point of your jetpack upgrade path. There is 5 tiers of jetpacks in total, and each tier requires a jetpack from the lower tier for crafting.` -> `Wooden Jetpack (Tier 0) is the starting point of your jetpack upgrade path. There are 5 tiers of jetpacks in total, and each tier requires a jetpack from the previous tier to craft.`